community/dnscrypt-proxy/dnscrypt-proxy.pre-deinstall

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+/etc/init.d/dnscrypt-proxy stop
+
+exit 0
```

Comment:
This should not be here. It’s admin’s responsibility to stop the service. Please remove it.